src/3d/symbols/qgspoint3dsymbol_p.cpp

```diff
@@ -469,20 +475,24 @@ void QgsModelPoint3DSymbolHandler::addMeshEntities( const Qgs3DMapSettings &map,
   // get nodes
   for ( const QVector3D &position : positions )
   {
-    // build the entity
-    Qt3DCore::QEntity *entity = new Qt3DCore::QEntity;
+    const QString source = QgsApplication::instance()->sourceCache()->localFilePath( symbol.shapeProperties()[QStringLiteral( "model" )].toString() );
+    if ( !source.isEmpty() )
```

Comment:
Could you please add a comment around here that if the source is remote, the Qgs3DMapScene will take care of refreshing this 3D symbol when the source is fetched? (so that it's clear we didn't give up, just passively waiting...)